src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CalculatorTest {
+    private static Long firstOperand;
+    private static Long secondOperand;
+    private static Character operation;
+    private static Long expected;
+    private static Long actual;
+
+    @Before
+    public void init() {
+        firstOperand = null;
+        secondOperand = null;
+        operation = null;
+        expected = null;
+        actual = null;
+    }
+
+    @Test
+    public void sum() {
+        operation = Character.valueOf('+');
+        for (int i = -10; i < 10; i += 2) {
+            firstOperand = Long.valueOf(i);
+            secondOperand = Long.valueOf(10 - i);
+            actual = Calculator.apply(firstOperand, operation, secondOperand);
+            expected = Long.sum(firstOperand, secondOperand);
+            Assert.assertEquals("Incorrect result", actual, expected);
```

Comment:
```suggestion
            Assert.assertEquals("Incorrect result", expected, actual);
```
the same in other methods